### ISSUE TITLE
C# text colors wrong

### ISSUE THREAD

**Issue by Compusa** [samdark/intellij-visual-studio-code-dark-plus#18]
### What the issue is?
The text color scheme in this plug is not similar to the VS Code Dark+ (default) color scheme. I really love this theme, and it would be top notch if we could solve this.

### Screenshots
Rider
<img width="700" alt="Rider" src="https://user-images.githubusercontent.com/11718906/75611454-f7081200-5b1a-11ea-943d-edb75d93879f.png">

VS Code
<img width="700" alt="VSCode" src="https://user-images.githubusercontent.com/11718906/75611442-df308e00-5b1a-11ea-8f04-f0e6b8d5c1f9.png">

### How would you fix it?

Don't really know how to fix it, any advice or help appreciated.

### Additional info

| Q                | A
| ---------------- | ---
| Plugin Version   | 1.9
| IDE and version  | Rider 2019.3
| Operating system | Mac OS

**Comment by samdark** [comment]
You can adjust the schema in Settings -> Editor -> Color Scheme -> C# (or whatever it's called). Then export it in `.icls`. That would give you XML to merge into https://github.com/samdark/intellij-visual-studio-code-dark-plus/blob/master/resources/visual_studio_code_dark_plus.xml

See https://github.com/samdark/intellij-visual-studio-code-dark-plus/pull/16

**Comment by samdark** [comment]
Overall you can't achieve exactly the same colors but some tweaks for specific language are definitely possible.

**Comment by Compusa** [comment]
Thank for the response. Yes, I was able to make som modifications. But I'm not able to differentiate keywords, i.e. not able to make `if` and `return` pink, for instance. I guess this is a limitation of the Rider IDE?

**Comment by samdark** [comment]
Yeah. It can't be *exactly* the same since Rider / IntelliJ has different set of things we are able to colorize than VS Code. But, at least, we can make it closer / better.

**Comment by samdark** [comment]
@Compusa in case you've adjusted the scheme for yourself, it would be great if you'll share the color scheme. See https://github.com/samdark/intellij-visual-studio-code-dark-plus/blob/master/.github/CONTRIBUTING.md#code-highlighting

**Comment by Code-Chops** [comment]
Here is my customized icls. It is more like Visual Studio. But maybe it is of any help :)

[Visual_Studio_Code_Dark_Plus.txt](https://github.com/samdark/intellij-visual-studio-code-dark-plus/files/9380627/Visual_Studio_Code_Dark_Plus.txt)

**Comment by samdark** [comment]
@Code-Chops do you mean visual studio or vscode?

**Comment by Code-Chops** [comment]
I mean VS (not code).
